### PR TITLE
Clean up latest release history

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by derekbekoe** [comment]
I'll make the change for other modules as well...

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -4,15 +4,11 @@ Release History
 ===============
 2.0.8 (2017-06-13)
 ++++++++++++++++++
-* Remove useless line-too-long suppression
 * fix acs kube get-credentials ssh-key loading (#3612)
 * Change a message so as not to confuse MacOS users. (#3568)
-* Eliminating too-many-arguments pylint disable rule (#3583)
-* Fix acs style
 * Expand service principal testing somewhat. (#3114)
 * Move all existing recording files to latest folder
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: Expand service principal testing and moving existing records can be removed.

File: src/command_modules/azure-cli-interactive/HISTORY.rst [hunk 2]
```diff
@@ -14,11 +14,8 @@ Release History
 * Fixes Interactive errors with the profile 2017-03-09-profile-preview (#3587)
 * Allows '--version' as a parameter for interactive mode (#3645)
 * Stop Interactive Mode from Throwing errors from Validation completions (#3570)
-* Fix various pylint disable rules
 * Progress Reporting for template deployments (#3510)
-* Fix method-hidden pylint disable rule
 * Improves the muting of parse args + appends history with gestures (#3434)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-interactive/HISTORY.rst]: I have no idea what this means.  Is it customer facing?  If so we should reword it or remove it.

File: src/command_modules/azure-cli-resource/HISTORY.rst [hunk 3]
```diff
@@ -12,12 +12,8 @@ unreleased
...
 ++++++++++++++++++
 * Fix up some parsing and error messages. (#3584)
-* Fix various pylint disable rules
 * Fix --resource-type parsing for the lock command to accept <resource-namespace>/<resource-type>
 * Add parameter checking for template link templates (#3629)
-* Eliminating too-many-arguments pylint disable rule (#3583)
-* Fix method-hidden pylint disable rule
-* Move all existing recording files to latest folder
 * Add support for specifying deployment parameters using KEY=VALUE syntax.
 * Add a bunch of help text. (#3582)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-resource/HISTORY.rst]: This one isn't helpful

File: src/command_modules/azure-cli-resource/HISTORY.rst [hunk 4]
```diff
@@ -12,12 +12,8 @@ unreleased
...
 * [ARM] Add KEY=VALUE syntax for deployment create (#3637)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-resource/HISTORY.rst]: This is the commit message for the more curated entry. I would remove this one.

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 5]
```diff
@@ -9,25 +9,16 @@ Release History
 
 2.0.8 (2017-06-13)
 ++++++++++++++++++
-* Remove useless line-too-long suppression
 * vm: support attaching data disks on vm create (#3644)
 * Improve table output for vm/vmss commands: get-instance-view, list, show, list-usage, etc
-* Fix all bad-continuation pylint disables
 * support configuring disk caching on attaching a managed disk (#3513)
 * core: Create subscription clients with right SDK profile (#3635)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Should this be in the VM module?

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 6]
```diff
@@ -9,25 +9,16 @@ Release History
...
-* Fix various pylint disable rules
 * Fixed _help.py for get-boot-log (#3616)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Help text changes shouldn't need to be in history.

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 7]
```diff
@@ -9,25 +9,16 @@ Release History
...
-* Eliminating too-many-arguments pylint disable rule (#3583)
 * output: add support for picking table output fields through jmespath query  (#3581)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Isn't this part of core?

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 8]
```diff
@@ -9,25 +9,16 @@ Release History
...
 * Support attaching existing data disks on vm create
-* Fix attribute-defined-outside-init pylint disable rules
-* Fix method-hidden pylint disable rule
-* Move all existing recording files to latest folder
 * fix typos in error message (#3643)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Would omit.

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 9]
```diff
@@ -9,25 +9,16 @@ Release History
...
-* Remove various pylint disable statements
 * VM/VMSS: fixed an issue with name generation that resulted in the create commands not being idempotent.
 * [VM/VMSS] Fix idempotency for VM/VMSS create (#3586)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Commit message version of curated message. Would remove.

**Comment by troydai** [comment]
Look what I did ...

File: src/command_modules/azure-cli-acs/HISTORY.rst (hunk 1 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: done

File: src/command_modules/azure-cli-interactive/HISTORY.rst (hunk 2 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-interactive/HISTORY.rst]: removed

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 9 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: done

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 8 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: done

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 7 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: done

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 6 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: done

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 5 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: done

File: src/command_modules/azure-cli-resource/HISTORY.rst (hunk 4 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-resource/HISTORY.rst]: done

File: src/command_modules/azure-cli-resource/HISTORY.rst (hunk 3 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-resource/HISTORY.rst]: done

**Comment by derekbekoe** [comment]
@tjprescott Look good to you?

**Pull request merged**